lab-justin/lib/bodyParser.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = function(req){
+  return new Promise(function(resolve, reject){
+    req.body = '';
+    req.on('data', function(data){
+      req.body +=data.toString();
+    });
+
+    req.on('end', function(){
+      try {
+        req.body = JSON.parse(req.body);
+        resolve();
```

Comment:
Call resolve with null here if you're calling reject with the error below. This indicates explicitly that you intended the error field to be falsy.